server/data/hmppsAuthClient.ts

```diff
@@ -19,10 +17,10 @@ function getSystemClientTokenFromHmppsAuth(username?: string): Promise<superagen
 
   const grantRequest = new URLSearchParams({
     grant_type: 'client_credentials',
-    ...(username && { username }),
+    username, // todo
   }).toString()
 
-  logger.info(`${grantRequest} HMPPS Auth request for client id '${config.apis.hmppsAuth.systemClientId}''`)
+  logger.info(`${grantRequest} HMPPS Auth request for client id. '${config.apis.hmppsAuth.systemClientId}''`)
```

Comment:
Why the double single quotes at the end?

Reply:
It was like this on main - I can tidy that.

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/136/commits/5fb51a21fa85dc9f0b8cc486751e895965630443